src/printer.ts

```diff
@@ -420,21 +416,24 @@ export class PugPrinter {
 			this.possibleClassPosition = this.result.length;
 			result = '(';
 			logger.debug(this.currentLineLength);
-			this.currentLineLength += 1;
```

Comment:
@Shinigami92 I was able to merge the counting of the opening parenthesis together with the closing one further down. This made the code there simpler too, because I don't need to remove the already counted one if there aren't any attributes.